lib/rules/web/admin_console/4.3/filters.xyaml

```diff
@@ -98,3 +98,6 @@ check_item_in_table:
   elements:
   - selector:
       xpath: //tr[@data-test-rows='resource-row']//*[text()='<item>']
+
+# Below is PF4 filter bar related rules (since OCP4.5)
+open_filter_dropdown_list: {}
```

Comment:
The comments stated that the rules exist since 4.5, but we are adding it to 4.3 ? Which one should be believe ?

Reply:
@liangxia both 4.3 and 4.4 are same (should use empty rule for open_filter_dropdown_list), 4.5 needs this rule to do an extra Click, the scenario apply to cluster >=4.3

Reply:
@liangxia So are we good to merge this PR? 
Thanks in advance!

Reply:
Sounds the comment should use 'since OCP4.3'

Reply:
@liangxia No, this rule does not needed at all for 4.3 and 4.4, since 4.5 needs an extra click, we always use empty same-named rule for previous versions. Otherwise, although 4.5 could PASS, 4.3 and 4.4 will failed if there is no same-named rule at all. If you search the empty rule using `{}` , you can find bunches of previous examples. Do I make it clear?

Reply:
If the rule does not needed at all for 4.3 and 4.4, then we should remove it from 4.3 and 4.4.
If it is needed, that mean it starts from OCP4.3.
I'm leaving it to @yapei to decide. I can merge once got her ack.

Reply:
@liangxia the rules added for 4.3 and 4.4 is empty, since the scenario script is used for all cluster versions, but page element would change during different versions, so need these empty rules for some versions to keep compatible, otherwise, the scenario will fail. I suggest to remove the comment "# Below is PF4 filter bar related rules (since OCP4.5)", which causes misunderstanding.

Reply:
ok. Will ask @yapei